fyle/platform/internals/list_resources.py

```diff
@@ -24,7 +24,7 @@ def list(self, query_params=None) -> Dict:
         :return: List of Resource Objects
         """
         query_params = {} if query_params is None else query_params
-        if not str(query_params.get('offset')) or not str(query_params.get('limit')) or not query_params.get('order'):
+        if query_params.get('offset') is None or query_params.get('limit') is None or not query_params.get('order'):
```

Comment:
Should we not do the same check for **order** too? 
**query_params.get('order') is None**

Reply:
yeah resolved